roboflow/adapters/rfapi.py

```diff
@@ -103,6 +102,7 @@ def save_annotation(
     annotation_name: str,
     annotation_string: str,
     image_id: str,
+    job_name: str = DEFAULT_JOB_NAME,
```

Comment:
should we add this to cmd arg also like batch name?

Reply:
I don't think so. 
IMO the user either wants to set the name in this box or not.
The distinction between batch and job is a complexity the user shouldn't care at this point.